### ISSUE TITLE
Pix2Pix ResUNet has weird coloured artifacts

### ISSUE THREAD

**Issue by ibrahim1331** [junyanz/pytorch-CycleGAN-and-pix2pix#1407]
Hi @junyanz,

I have tried to train the Pix2Pix model to on the sat2map dataset using the ResUNet Generator, however, I find that when training the ResUNet Generator, the output has weird coloured artifacts on the images throughout the training and also on the validation set. I also find that the Discriminator loss gets very low (close to zero) after the first few epochs and remains low except for a few spikes.

Here are some sample images of the generated images from the validation set.

![image](https://user-images.githubusercontent.com/17927036/161936846-18bb8d86-3fd1-49e5-afe9-e26aeb61dd6c.png)
![image](https://user-images.githubusercontent.com/17927036/161936924-59a809cc-139e-4f13-b2e8-c2bd924b3d67.png)
![image](https://user-images.githubusercontent.com/17927036/161936972-f832770a-ae10-4397-822c-2c30c1fd855a.png)

I would like to ask if there is a fix to this problem.

Thanks.
